lib/ecto/postgres/pg_framestamp_migrations.ex

```diff
@@ -1082,6 +1283,68 @@ defpgmodule Vtc.Ecto.Postgres.PgFramestamp.Migrations do
     end
   end
 
+  ### SQL fragment builders
+
+  # Returns declaration list for comparison operators,
+  @spec compare_declarations() :: Postgres.Utils.function_declarations()
+  defp compare_declarations do
+    [
+      a_cmp: {:bigint, "((a).__seconds_n * (b).__seconds_d)"},
+      b_cmp: {:bigint, "((b).__seconds_n * (a).__seconds_d)"},
+      cmp_sign: {:bigint, "SIGN(a_cmp - b_cmp)"}
+    ]
+  end
+
+  # Returns subtraction declarations.
+  @spec addition_declares() :: Postgres.Utils.function_declarations()
+  defp addition_declares do
+    [
+      numerator: {:bigint, "((a).__seconds_n * (b).__seconds_d) + ((b).__seconds_n * (a).__seconds_d)"},
+      denominator: {:bigint, "(a).__seconds_d * (b).__seconds_d"},
+      greatest_denom: {:bigint, "GCD(numerator, denominator)"}
+    ]
+  end
+
+  # Returns subtraction declarations.
+  @spec subtraction_declares() :: Postgres.Utils.function_declarations()
+  defp subtraction_declares do
+    [
+      numerator: {:bigint, "((a).__seconds_n * (b).__seconds_d) - ((b).__seconds_n * (a).__seconds_d)"},
+      denominator: {:bigint, "(a).__seconds_d * (b).__seconds_d"},
+      greatest_denom: {:bigint, "GCD(numerator, denominator)"}
+    ]
+  end
+
+  # Returns SQL statement to rause mixed framerate arithmatic operator.
+  @spec sq_raise_mised_frame_arithmetic_error(atom(), atom()) :: raw_sql()
+  defp sq_raise_mised_frame_arithmetic_error(type, operator) do
```

Comment:
per above note, i believe this is supposed to be:
```suggestion
  @spec sql_raise_missed_frame_arithmetic_error(atom(), atom()) :: raw_sql()
  defp sql_raise_missed_frame_arithmetic_error(type, operator) do
```

Reply:
It's supposed to be `sql_raise_mixeed_frame_arithmetic_error`. It raises when you are doing a mixed frame calculation that returns a Framestamp,  but haven't used one of the mixed rate operators that indicate which side's framerate to adhere to.